### PR TITLE
Fixed side nav(#207)

### PR DESCRIPTION
**Pull request by BhupaliRane**
fixed sidenav(Issue #207).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
 	"dependencies": {
 		"animate.css": "^4.1.1",
 		"bulma": "^0.9.1",
-		"bulma-dracula": "^1.1.1"
+		"bulma-dracula": "^1.1.1",
+		"pnpm": "^5.8.0"
 	},
```
Comment (mazipan) [inline package.json:23]: Can you remove this `pnpm` dependency?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (mazipan) [inline package-lock.json:1]: Remove this lock file, we use `pnpm` instead of traditional `npm`.
So it doesn't need this file.

**Review by mazipan — CHANGES_REQUESTED** [review]

**Comment by mazipan** [comment]
Also please help to attach the screenshot on the PR.

**Comment by BhupaliRane** [comment]
Done with the changes
Screenshot :
Screenshot (592)

**Comment by mazipan** [comment]
Remove `package-lock` and revert changes on `pnpm-lockfile`.
It make CI failed

**Comment by BhupaliRane** [comment]
Done

**Review by mazipan — APPROVED** [review]

**Pull request merged**